logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/MessageRepositoryExtensions.kt

```diff
@@ -50,6 +52,12 @@ internal interface MessageRepositoryExtensions {
         pagingConfig: PagingConfig,
         startingOffset: Long
     ): Flow<PagingData<Message.Standalone>>
+
+    suspend fun getPaginatedMessageAssetImageByConversationId(
```

Comment:
as far as it returns `Flow` - `observePaginatedMessageAssetImageByConversationId` fits better, imo